docs/reference/ig.md

```diff
@@ -318,7 +318,7 @@ List of flags:
 
 ### Using ig in a Kubernetes pod
 
-In order to run `ig` in a Kubernetes pod use [examples/pod-ig.yaml](examples/pod-ig.yaml).
+In order to run `ig` in a Kubernetes pod use [examples/pod-ig.yaml](../examples/pod-ig.yaml).
```

Comment:
Ah, I think it's again https://github.com/inspektor-gadget/inspektor-gadget/issues/2254. That change doesn't fix the issue. I think we need to use absolute URLs all around for things in that folder. Let me check it.

Reply:
Added a couple of commits that hopefully improve it.